ElasticBox/appliance-overview.md

```diff
@@ -7,15 +7,43 @@
 }}}
 
 ### ElasticBox Enterprise Appliance Overview
+<<<<<<< HEAD
+<<<<<<< HEAD
+Run ElasticBox on-premise as a virtual appliance in your vCenter datacenter or OpenStack cloud. After you install and [set up ElasticBox](./appliance-initialsetup.md) for use, you’re ready to automate deployments to any cloud and platform.
+
+**Try the Appliance for Free**
+The appliance is available as a free download. Experience all the same functionality as the paid Enterprise Edition, except for advanced features like managing your ElasticBox organization, sharing and monitoring resources, and deploying to private clouds. To get these features, upgrade to the Enterprise Edition.
+=======
 Run ElasticBox on-premise as a virtual appliance in your vCenter datacenter or OpenStack cloud. After you install and [set up ElasticBox](../ElasticBox/appliance-initialsetup.md) for use, you’re ready to automate deployments to any cloud and platform.
+=======
+Run ElasticBox on-premise as a virtual appliance in your vCenter datacenter or OpenStack cloud. After you install and [set up ElasticBox](../ElasticBox/appliance-initialsetup.md) for use, you’re ready to automate deployments to any cloud and platform.
+
+**Try the Appliance for Free**
+The appliance is available as a free download. Experience all the same functionality as the paid Enterprise Edition, except for advanced features like managing your ElasticBox organization, sharing and monitoring resources, and deploying to private clouds. To get these features, upgrade to the Enterprise Edition. To learn more, see [ElasticBox Editions](../ElasticBox/about-elasticbox-editions.md).
+>>>>>>> CenturyLinkCloud/master
 
 **Try the Appliance for Free**
 The appliance is available as a free download. Experience all the same functionality as the paid Enterprise Edition, except for advanced features like managing your ElasticBox organization, sharing and monitoring resources, and deploying to private clouds. To get these features, upgrade to the Enterprise Edition. To learn more, see [ElasticBox Editions](../ElasticBox/about-elasticbox-editions.md).
 
+>>>>>>> CenturyLinkCloud/master
 **Download the ElasticBox Virtual Appliance**
 Request a download link from [support](mailto:support@elasticbox.com).
 
 ### Next Up
+<<<<<<< HEAD
```

Comment:
Clean up this section. remove the `<<<<<<< HEAD`, `=======`, and `>>>>>>> CenturyLinkCloud/master` lines. Also remove the duplicate content.